lxd/daemon.go

```diff
@@ -765,15 +765,15 @@ func (d *Daemon) Init() error {
 		return err
 	}
 
-	// 
+	//
 	// API addition: storage
 	// Repository for storage pool loop files when the user has not
 	// specified a source.
 	if err := os.MkdirAll(shared.VarPath("disks"), 0700); err != nil {
 		return err
 	}
 
-	// 
+	//
 	// API addition: storage
 	// Mountpoints for storage pools.
 	if err := os.MkdirAll(shared.VarPath("storage-pools"), 0711); err != nil {
```

Comment:
So I don't think we care when those two were introduced, just like we didn't when network was introduced.

I'd just merge those two MkdirAll with the rest of the others above which makes it easier to get a glance of all the directory permissions in one shot.